src/vectors.S

```diff
@@ -0,0 +1,112 @@
+.section .text.init
+.global _run_guest_vector,_guest_exit_vector
+
+// t5 -> cached sstatus
+// Restore vectors for guest
+// i.e. memory to main state
+.align 2
+_run_guest_vector:
+    
+    // Save sstatus
+    csrr t5, sstatus
+
+    // Enable vector commands
+    addi t4, zero, {sstatus_vs_enable}
```

Comment:
nit: Use `li` pseudo-instruction to be more concise.

Reply:
will fix